helix-common/pom.xml

```diff
@@ -125,7 +125,7 @@
         <version>3.3.0</version>
         <executions>
           <execution>
-            <id>default-package-jdk11</id>
```

Comment:
Why we make jdk8 as default package

Reply:
oh yes this is incorrect.. I will revert.